src/Component/Filtering/AbstractFilter.php

```diff
@@ -0,0 +1,18 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Paweł Jędrzejewski
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+namespace Sylius\Component\Grid\Filtering;
+
+abstract class AbstractFilter implements FilterInterface, TypeAwareFilterInterface, FormTypeAwareFilterInterface
```

Comment:
What would you say about the following change in the naming?
```suggestion
interface ConfigurableFilterInterface implements FilterInterface, TypeAwareFilterInterface, FormTypeAwareFilterInterface
```

Reply:
Great Idea ! Indeed that's better than thé abstract class...